content/en/entities/Suggestion.md

```diff
@@ -36,7 +36,20 @@ aliases: [
 `past_interactions` = You have interacted with this account previously\
 `global` = This account has many reblogs, favourites, and active local followers within the last 30 days\
 **Version history:**\
-3.4.0 - added
+3.4.0 - added\
+4.3.0 - deprecated, use `sources` instead
```

Comment:
question / comment about deprecated - will `source` continue to be supported for n releases, or is this a hard replacement in 4.3.0?

Reply:
`source` is still supported for compatibility, there is no timeline as when it will be removed yet